capstone/src/main/java/com/google/step/data/RestaurantScore.java

```diff
@@ -0,0 +1,46 @@
+package com.google.step.data;
+
+/**
+ * Class that holds the score for a specific restaurant.
+ */
+public class RestaurantScore {
+
+    // The name should always remain the same but the score will change when calculating it.
+    private final String restaurantName;
+    private double score;
+
+    /**
+     * Constructor for the class.
+     * @param restaurantName name of the restaurant.
+     * @param score for the restaurant.
+     */
+    public RestaurantScore(String restaurantName, double score) {
+        this.restaurantName = restaurantName;
+        this.score = score;
+    }
+
+    /**
+     * Get the restaurant Name.
+     * @return String representing the restaurant name.
+     */
+    public String getRestaurantName() {
+        return restaurantName;
+    }
+
+    /**
+     * Get the score for the restaurant.
+     * @return double representing the score.
+     */
+    public double getScore() {
+        return score;
+    }
+
+    /**
+     * Set the score to a new value.
+     * @param score double representing the new score.
+     */
+    public void setScore(double score) {
```

Comment:
Will you be needing this setter at some point? It seems unnecessary.

Reply:
Yes Because I'm changing the score as it goes through the different standardization and normalization calculations.